ingestion_tools/scripts/importers/identified_object.py

```diff
@@ -0,0 +1,118 @@
+
+from typing import Any
+
+import pandas as pd
+
+from common.config import DepositionImportConfig
+from common.finders import DefaultImporterFactory
+from common.id_helper import IdentifierHelper
+from common.metadata import IdentifiedObjectMetadata
+from importers.base_importer import BaseFileImporter
+
+
+class IdentifiedObjectIdentifierHelper(IdentifierHelper):
+    @classmethod
+    def _get_container_key(cls, config: DepositionImportConfig, parents: dict[str, Any], *args, **kwargs) -> str:
+        return "-".join(["identified_object", parents["run"].get_output_path()])
```

Comment:
Is the order of the elements in the list correct? 
I believe this would produce `identified_object/{bucket_prefix}/{dataset_id}/{run_name}/`, and we want: 
`{bucket_prefix}/{dataset_id}/{run_name}/identified_object/`.